client/repoutils.c

```diff
@@ -276,6 +281,156 @@ TDNFRepoRemoveCache(
     goto cleanup;
 }
 
+uint32_t
+TDNFRemoveRpmCache(
```

Comment:
Just a thought. Should we make this into a helper routine TDNFRemoveDirRecursive and have TDNFRemoveRpmCache call it? If there are other places where we'd need this?

Reply:
Only in this function we require this, other remove metadata and solvcache has its own remove functionalities.